test/BUILD.bazel

```diff
@@ -238,7 +238,7 @@ cc_test(
             # glog CHECK die with a fatal error
             "bthread_key_unittest.cpp",
             "bthread_butex_multi_tag_unittest.cpp",
-            "bthread_rwlock_unittest.cpp",
+            #"bthread_rwlock_unittest.cpp",
```

Comment:
Is there any specific reason why this line is commented out?

Reply:
@shanhe72101 It appears that the bthread_rwlock_unittest.cpp file originally existed here, but your PR has commented out this test file. Is this as expected?